<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Phonebook</class>
 <widget class="QWidget" name="Phonebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phonebook</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>401</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>            电话</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_Qphone"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btn_Qname">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Qphone">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Btn_new">
      <property name="text">
       <string>新建</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_delete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_logout">
      <property name="text">
       <string>注销</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_import">
      <property name="text">
       <string>导入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_export">
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>            姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_Qname"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
